app/js/components/pages/Job.jsx

```diff
@@ -64,26 +72,32 @@ const Flow = ({ plugin, workflow, onClickSubmit, onClickCancel }) => {
             )}
             </form>
             <button
-                className="btn btn-danger"
-                onClick={onClickCancel}
+                className="btn btn-primary"
+                onClick={onClickSubmit}
             >
-                Cancel
+                Go!
             </button>
             <button
-                className="btn btn-primary pull-right"
-                onClick={onClickSubmit}
+                className="btn btn-danger pull-right"
+                onClick={onClickCancel}
             >
-                Go!
+                Cancel
             </button>
         </div>
   );};
 
-Flow.propTypes = {
+Job.propTypes = {
+    inputArtifacts: React.PropTypes.object,
     plugin: React.PropTypes.object,
     workflow: React.PropTypes.object,
     router: React.PropTypes.object,
     onClickSubmit: React.PropTypes.func,
     onClickCancel: React.PropTypes.func
 };
 
-export default Flow;
+Job.contextTypes = {
```

Comment:
Oops, these were leftovers from trying to figure out how to get the artifacts loaded on a per-page basis.